pkg/virt-controller/services/template.go

```diff
@@ -680,6 +680,12 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 		}
 	}
 
+	if util.IsGpuVmi(vmi) {
```

Comment:
Let's just check if `vmi.Spec.Domain.Devices.Gpus` isn't empty. I don't think util.IsGpuVmi is required.